v2/paths/serial.yaml

```diff
@@ -0,0 +1,135 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+get:
+  tags:
+    - OpenAccess
+    - Synchronous
+  summary: Get the current serial assertion
+  description: |-
+    Retrieves the current serial assertion for the system. The serial assertion
+    is a statement used to bind a device identity to it's public key, provided by the store.
+  externalDocs:
+    description: Read more about serial assertions on the Ubuntu Core documentation.
+    url: https://documentation.ubuntu.com/core/reference/assertions/serial/
+  operationId: getSerialAssertion
+  security: []
+  responses:
+    200:
+      description: The raw serial assertion text.
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: |
+              type: serial
+              authority-id: generic
+              brand-id: generic
+              model: generic-classic
+              serial: 46923e6d-5d45-420d-905a-99a9e92493b4
+              device-key:
+                AcbBTQRWhcGAARAA...
+                ...AEQEAAQ==
+              device-key-sha3-384: PznqOqWAx4_f8tFafGI2...
+              timestamp: 2025-07-17T03:11:33.518427Z
+              sign-key-sha3-384: wrfougkz3Huq2T_Kklfnu...
+              ...bX5JkJG5cunW0h/
+    404:
+      $ref: '../components/responses/NotFound.yaml'
+
+post:
+  tags:
+    - Asynchronous
+    - AuthenticationRequired
+  summary: Perform an action on the serial assertion
+  description: |-
+    Forgets the existing serial assertion, causing the system to pick up a new
+    one. This endpoint uses a custom payload format and can optionally include a
+    JSON body with specific actions.
+  externalDocs:
+    description: Read more about offline remodelling on the Ubuntu Core documentation.
+    url: https://documentation.ubuntu.com/core/explanation/remodelling/index.html#heading--offline
+  operationId: setSerialAssertion
+  security:     # TODO document POST request properly
+    - PeerAuth: []
+  requestBody:
```

Comment:
I am quite confused about requestBody, afaiu a POST /v2/model/serial will have as body just some json with two possible fields, "action" and "no-registration-until-reboot"